hack/bootstrap-buildx

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+if [ "$#" -ne 1 ]; then
+  echo >&2 "usage: $0 <name>"
+  exit 1
+fi
+NAME="$1"
+echo "Bootstrapping buildx builder \"${NAME}\""
+
+topdir="$(realpath "$(dirname "$0")/..")"
+cd "${topdir}"
+buildx="./bin/buildx"
+if [ ! -f "${HOME}/.docker/buildx/instances/${NAME}" ]; then
+  set -x
+  "${buildx}" create --name "${NAME}"
```

Comment:
```suggestion
  "${buildx}" create --driver docker-container --name "${NAME}"
```

Maybe use `docker-container` driver?

Reply:
It should be the default, but wouldn't hurt to specify docker-container explicitly. Updated.